incydr/cli/cmds/cases.py

```diff
@@ -223,15 +208,21 @@ def update(
         )
 
     client = ctx.obj()
-    data = UpdateCaseRequest(
-        assignee=assignee,
-        description=description,
-        findings=findings,
-        name=name,
-        status=status,
-        subject=subject,
-    )
-    client.session.put(f"/v1/cases/{case_number}", json=data.dict())
+    case = client.cases.v1.get_case(case_number)
+    if assignee:
+        case.assignee = user_lookup(client, assignee)
+    if description:
+        case.description = description
+    if findings:
+        case.findings = findings
+    if name:
+        case.name = name
+    if status:
+        case.status = status
+    if subject:
+        case.subject = user_lookup(client, subject)
```

Comment:
the user lookup is already done for `subject` and `assignee` within the callback on the `click.option` definition